Broken ocaml-manual package in opam 2

I upgraded last week to opam 2 (on a mac) and I was not able to install ocaml-manual package even from a clean empty opam directory.

I get
```
[ERROR] The compilation of ocaml-manual failed at "/Users/fdagnat/.opam/opam-init/hooks/sandbox.sh build cp -R .
        /Users/fdagnat/.opam/default/doc/ocaml-manual".

#=== ERROR while compiling ocaml-manual.4.07.0 ================================#
# context     2.0.0 | macos/x86_64 | ocaml-system.4.07.0 | https://opam.ocaml.org#eeb7eb22
# path        ~/.opam/default/.opam-switch/build/ocaml-manual.4.07.0
# command     ~/.opam/opam-init/hooks/sandbox.sh build cp -R . /Users/fdagnat/.opam/default/doc/ocaml-manual
# exit-code   1
# env-file    ~/.opam/log/ocaml-manual-8703-ffb3fd.env
# output-file ~/.opam/log/ocaml-manual-8703-ffb3fd.out
### output ###
# [...]
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./runtime.html: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./const.html: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./ocamldoc.html: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./previous_motif.svg: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./debugger.html: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./libgraph.html: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./libbigarray.html: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./manual033.html: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./libthreads.html: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./contents_motif.svg: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./libnum.html: Operation not permitted
# cp: /Users/fdagnat/.opam/default/doc/ocaml-manual/./compunit.html: Operation not permitted
```

From my experiments, ocaml-manual use a build clause in the opam package to do the cp from build directory to the actual location but this raise an error:
```
#cd /Users/fdagnat/.opam/default/.opam-switch/build/ocaml-manual.4.07.0/
#/Users/fdagnat/.opam/opam-init/hooks/sandbox.sh build cp -R . /Users/fdagnat/.opam/default/doc/ocaml-manual
cp: /Users/fdagnat/.opam/default/doc/ocaml-manual: Operation not permitted
…
```

I do not fully understand what sandbox.sh does but it seems that build command switch the rights of the opam dir to ro…
Using install instead of build worked for me:
```
/Users/fdagnat/.opam/opam-init/hooks/sandbox.sh install cp -R . /Users/fdagnat/.opam/default/doc/ocaml-manual
```

--- a/packages/ocaml-manual/ocaml-manual.4.01.0/opam
+++ b/packages/ocaml-manual/ocaml-manual.4.01.0/opam
@@ -11,7 +11,7 @@ doc: "http://caml.inria.fr/pub/docs/manual-ocaml/"
 license: "(c) Institut National de Recherche en Informatique et en Automatique (INRIA)"
 dev-repo: "git+https://github.com/ocaml/ocaml.git"
 bug-reports: "http://caml.inria.fr/mantis/"
-build: [
+install: [
  [ "cp" "-R" "." ocaml-manual:doc ] {os != "win32"}
  [ "robocopy" "/E" "." ocaml-manual:doc ] {os = "win32"}
 ]

--- a/packages/ocaml-manual/ocaml-manual.4.02.0/opam
+++ b/packages/ocaml-manual/ocaml-manual.4.02.0/opam
@@ -11,7 +11,7 @@ doc: "http://caml.inria.fr/pub/docs/manual-ocaml/"
 license: "(c) Institut National de Recherche en Informatique et en Automatique (INRIA)"
 dev-repo: "git+https://github.com/ocaml/ocaml.git"
 bug-reports: "http://caml.inria.fr/mantis/"
-build:
+install:
 [
  [ "cp" "-R" "." ocaml-manual:doc ] {os != "win32"}
  [ "robocopy" "/E" "." ocaml-manual:doc ] {os = "win32"}

--- a/packages/ocaml-manual/ocaml-manual.4.03.0/opam
+++ b/packages/ocaml-manual/ocaml-manual.4.03.0/opam
@@ -11,7 +11,7 @@ doc: "http://caml.inria.fr/pub/docs/manual-ocaml/"
 license: "(c) Institut National de Recherche en Informatique et en Automatique (INRIA)"
 dev-repo: "git+https://github.com/ocaml/ocaml.git"
 bug-reports: "http://caml.inria.fr/mantis/"
-build:
+install:
 [
  [ "cp" "-R" "." ocaml-manual:doc ] {os != "win32"}
  [ "robocopy" "/E" "." ocaml-manual:doc ] {os = "win32"}

--- a/packages/ocaml-manual/ocaml-manual.4.04.0/opam
+++ b/packages/ocaml-manual/ocaml-manual.4.04.0/opam
@@ -11,7 +11,7 @@ doc: "http://caml.inria.fr/pub/docs/manual-ocaml/"
 license: "(c) Institut National de Recherche en Informatique et en Automatique (INRIA)"
 dev-repo: "git+https://github.com/ocaml/ocaml.git"
 bug-reports: "http://caml.inria.fr/mantis/"
-build:
+install:
 [
  [ "cp" "-R" "." ocaml-manual:doc ] {os != "win32"}
  [ "robocopy" "/E" "." ocaml-manual:doc ] {os = "win32"}

--- a/packages/ocaml-manual/ocaml-manual.4.05.0/opam
+++ b/packages/ocaml-manual/ocaml-manual.4.05.0/opam
@@ -11,7 +11,7 @@ doc: "http://caml.inria.fr/pub/docs/manual-ocaml/"
 license: "(c) Institut National de Recherche en Informatique et en Automatique (INRIA)"
 dev-repo: "git+https://github.com/ocaml/ocaml.git"
 bug-reports: "http://caml.inria.fr/mantis/"
-build:
+install:
 [
  [ "cp" "-R" "." ocaml-manual:doc ] {os != "win32"}
  [ "robocopy" "/E" "." ocaml-manual:doc ] {os = "win32"}

--- a/packages/ocaml-manual/ocaml-manual.4.05.0/opam
+++ b/packages/ocaml-manual/ocaml-manual.4.05.0/opam
@@ -23,5 +23,5 @@ depends: [
 url {
   src:
     "http://caml.inria.fr/distrib/ocaml-4.05/ocaml-4.05-refman-html.tar.gz"
-  checksum: "md5=1649c91a84fd35444192d6a55f932939"
+  checksum: "md5=724ac6f96f019999cb83d95adafdda8e"
 }

--- a/packages/ocaml-manual/ocaml-manual.4.06.0/opam
+++ b/packages/ocaml-manual/ocaml-manual.4.06.0/opam
@@ -11,7 +11,7 @@ doc: "http://caml.inria.fr/pub/docs/manual-ocaml/"
 license: "(c) Institut National de Recherche en Informatique et en Automatique (INRIA)"
 dev-repo: "git+https://github.com/ocaml/ocaml.git"
 bug-reports: "http://caml.inria.fr/mantis/"
-build:
+install:
 [
  [ "cp" "-R" "." ocaml-manual:doc ] {os != "win32"}
  [ "robocopy" "/E" "." ocaml-manual:doc ] {os = "win32"}

--- a/packages/ocaml-manual/ocaml-manual.4.07.0/opam
+++ b/packages/ocaml-manual/ocaml-manual.4.07.0/opam
@@ -11,7 +11,7 @@ doc: "http://caml.inria.fr/pub/docs/manual-ocaml/"
 license: "(c) Institut National de Recherche en Informatique et en Automatique (INRIA)"
 dev-repo: "git+https://github.com/ocaml/ocaml.git"
 bug-reports: "http://caml.inria.fr/mantis/"
-build:
+install:
 [
  [ "cp" "-R" "." ocaml-manual:doc ] {os != "win32"}
  [ "robocopy" "/E" "." ocaml-manual:doc ] {os = "win32"}

